src/topology/instances/ennreal.lean

```diff
@@ -950,6 +950,27 @@ begin
   exact tsum_bUnion_le _ _ _
 end
 
+lemma tsum_le_of_sum_range_le {f : ℕ → ℝ≥0∞} {c : ℝ≥0∞}
+  (h : ∀ n, ∑ i in finset.range n, f i ≤ c) :
+  ∑' n, f n ≤ c :=
+le_of_tendsto' ((ennreal.has_sum_iff_tendsto_nat _).1 ennreal.summable.has_sum) h
```

Comment:
I think this is true in more generality than ennreal, maybe with the restriction of f nonneg?

Reply:
Yeah, I've noticed there are two existing `tsum_le_of_sum_range_le` for reals and nnreals already. I'll see if I can generalize them into a single lemma once I have cache

Reply:
#16951